aiida/manage/tests/pytest_fixtures.py

```diff
@@ -133,13 +133,17 @@ def test_1(aiida_local_code_factory):
     :rtype: object
     """
 
-    def get_code(entry_point, executable, computer=aiida_localhost):
+    def get_code(entry_point, executable, computer=aiida_localhost, prepend_text=None, append_text=None):
         """Get local code.
         Sets up code for given entry point on given computer.
 
         :param entry_point: Entry point of calculation plugin
         :param executable: name of executable; will be searched for in local system PATH.
         :param computer: (local) AiiDA computer
+        :param prepend: a string of code that will be put in the scheduler script before the
```

Comment:
Please change it in the docstring also.

Reply:
Sorry my bad. I have overlooked it. Done.